awxkit/setup.py

```diff
@@ -67,7 +67,7 @@ def run(self):
     },
     include_package_data=True,
     install_requires=requirements,
-    python_requires=">= 3.5",
+    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*",
```

Comment:
:nauseated_face: 

(not that I have a better suggestion)

Reply:
Yea, from reading the related PEP, this is *apparently* the official blessed way to do this.

Gross.